### ISSUE TITLE
[Suggestion] Text size

### ISSUE THREAD

**Issue by Liyf4TakingLs** [TekExplorer/featherdactyl.com#39]
**For reference, what is the current latest app version?**
```
Version: v0.0.22
```

Currently the text is too big on some parts of the app. (Example: File Editor, cuts after 20-ish char.)

**Describe the solution you'd like**
An option that allows you to change the text size of specific parts

**Describe alternatives you've considered**
Per phone style configuration, but could be annoying to implement. 

**Additional context**
People which have worse eyesight could make their text even bigger. People which like developing on their phones could make it more easy to see the whole text.

**Comment by drjayy** [comment]
@Liyf4TakingLs the screenshot you provided, doesn't look like the intended design choice by Tek

**Comment by Liyf4TakingLs** [comment]
Yeah, would also be nice if we could define the size in the configuration, like a slider from 8px to 32px (example)
